argocd/base/argo-cd-webhook-ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: argo-cd-webhook
+spec:
+  rules:
+    - host: cd.apps.argoproj.io
+      http:
+        paths:
+          - path: /api/webhook
```

Comment:
This will conflict with the server `/api/webhook` path. Since the server has the same path. You should use `nginx.ingress.kubernetes.io/rewrite-target: /api/webhook` since we use nginx in the demo app, and expose `/appset/api/webhook`.